src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -21,32 +22,59 @@
 public class SampleDataUtil {
     public static Person[] getSamplePersons() {
         try {
-            return new Person[] {
-                new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
+            Person alex = new Person(
+                    new Name("Alex Yeoh"),
+                    new Phone("87438807"),
+                    new Email("alexyeoh@example.com"),
                     new Address("Blk 30 Geylang Street 29, #06-40"),
                     new Favorite(true),
-                    getTagSet("friends")),
-                new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
+                    getTagSet("friends"),
+                    getSocialInfoSet(
+                            new SocialInfo("FACEBOOK", "alex", "https://facebook.com/alex")));
```

Comment:
Should social info type i.e. `FACEBOOK` be in lowercase?
Because your `SocialInfoMapping` parser does not allow uppercase.

This would cause an inconsistency in the display of person cards.